README.md

```diff
@@ -199,6 +199,10 @@ example, here's how to send a user a direct message only if they already follow
 
 #### Search Tweets in your timeline that match a specified query
     t search timeline "query"
+**Note**: In Twitter API parlance, your “home timeline” is your “Newsfeed” whereas your “user timeline” are the tweets tweeted (and retweeted) by you. 
+
+#### Search your own Tweets that match a specific query
+    t search timeline @yourtwitterhandle "query"
 
 #### Search Tweets in another user's timeline that match a specified query
     t search timeline @sferik "query"
```

Comment:
This is essentially the same command as the one you added. Perhaps they could be combined under a single description that makes it clearer. For example, instead of saying “Search Tweets in another user's timeline…” maybe say “Search Tweets in the specified user’s timeline…” What do you think about that?